src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -471,9 +472,17 @@ static bool _processCommand(Array<PathCommand>* cmds, Array<Point>* pts, char cm
         }
         case 'a':
         case 'A': {
-            _pathAppendArcTo(cmds, pts, cur, curCtl, arr[5], arr[6], arr[0], arr[1], arr[2], arr[3], arr[4]);
-            *cur = *curCtl = {arr[5], arr[6]};
-            *isQuadratic = false;
+            if (arr[0] < FLT_EPSILON || arr[1] < FLT_EPSILON) {
```

Comment:
Check the absolute values.

fabsf ...

Reply:
Thank you for your review. I modified code.